librad/src/git.rs

```diff
@@ -42,72 +43,39 @@ const CONTRIBUTOR_METADATA_FILE: &str = "contributor.json";
 
 const RAD_REMOTE_NAME: &str = "rad";
 
-#[derive(Debug, Fail)]
+#[derive(Debug, Error)]
 pub enum Error {
-    #[fail(display = "Invalid PGP key: missing UserID packet")]
+    #[error("Invalid PGP key: missing UserID packet")]
     MissingPgpUserId,
 
-    #[fail(display = "Invalid PGP key: missing address")]
+    #[error("Invalid PGP key: missing address")]
     MissingPgpAddr,
 
-    #[fail(display = "Project already exists")]
+    #[error("Project already exists")]
```

Comment:
Would it help users to know which project doesn't exist?

Reply:
Yaahaha, that variant doesn’t actually seem to be used anymore :D

Reply:
:fire: